DotNet/Normalization/Listeners/ResourceAcquiredListener.cs

```diff
@@ -119,6 +125,30 @@ private async System.Threading.Tasks.Task StartConsumerLoop(CancellationToken ca
                             throw new DeadLetterException("Failed to extract FacilityId and CorrelationId from message.", ex);
                         }
 
+                        if (message.Message.Value.AcquisitionComplete && message.Message.Value.Resource == null)
+                        {
+                            _logger.LogInformation("Acquisition Complete tail message received. Producing message for measure eval.");
+                            var headers = new Headers
+                            {
+                                new Header(NormalizationConstants.HeaderNames.CorrelationId, Encoding.UTF8.GetBytes(messageMetaData.correlationId))
+                            };
+                            var resourceNormalizedMessage = new ResourceNormalizedMessage
+                            {
+                                AcquisitionComplete = message.Message.Value.AcquisitionComplete,
+                                PatientId = message.Message.Value.PatientId ?? "",
+                                QueryType = message.Message.Value.QueryType,
+                                ScheduledReports = message.Message.Value.ScheduledReports
+                            };
+                            Message<string, ResourceNormalizedMessage> produceMessage = new Message<string, ResourceNormalizedMessage>
+                            {
+                                Key = messageMetaData.facilityId,
+                                Headers = headers,
+                                Value = resourceNormalizedMessage
+                            };
+                            await kafkaProducer.ProduceAsync(KafkaTopic.ResourceNormalized.ToString(), produceMessage);
+                            return;
+                        }
+
```

Comment:
Should we validate facilityId before sending the message out?

Reply:
sure. change made.